pnpm-lock.yaml

```diff
@@ -4923,7 +4923,7 @@ packages:
     engines: {node: ^10.13.0 || ^12.13.0 || ^14.15.0 || >=15.0.0}
     dependencies:
       '@jest/types': 27.5.1
-      '@types/node': 18.11.18
+      '@types/node': 18.15.11
```

Comment:
I'm not sure if these root level pnpm changes were intentional.

Reply:
Hmm. I guess not. I'll revert these and update. Thanks for noticing them!